src/schoology-extractor/schoology_extractor/helpers/arg_parser.py

```diff
@@ -73,9 +95,17 @@ def parse_main_arguments(args_in: list) -> MainArguments:
         help='The log level for the tool.',
         choices=constants.LOG_LEVELS,
         type=str,
-        default="INFO")
+        default="INFO",
+        env_var="LOG_LEVEL")
```

Comment:
Shouldn't this be `SCHOOLOGY_LOG_LEVEL`?
```suggestion
        env_var="SCHOOLOGY_LOG_LEVEL")
```

Reply:
✅